### DISCUSSION TITLE
树莓派可以用吗？

### DISCUSSION THREAD

**Discussion by arvidwen** [messense/aliyundrive-webdav#115] [Q&A] [answered]

**Comment by messense** [comment, marked as answer]
可以，直接下载二进制包，或者 `pip3 install aliyundrive-webdav`，按照 README 中命令行运行方式运行。

**Comment by messense** [comment]
也可以从 snapcraft 安装：https://snapcraft.io/install/aliyundrive-webdav/raspbian#install
